db/errorHandlers/errorHandlers.js

```diff
@@ -0,0 +1,5 @@
+
+
+exports.handle404 = (err, req, res, next) => {
+    return res.status(err.status).send({msg: err.msg})
```

Comment:
This handle404 is being mixed up with a custom error handler, which we don't quite need yet.
This function should be called when the request has a non-existent endpoint, (using app.all) and then send a 404 and a suitable message